### ISSUE TITLE
Lunr.jp failed to load at v2.1.1

### ISSUE THREAD

**Issue by cxlwill** [squidfunk/mkdocs-material#605]
## Description

Just upgrade material from v1.1.2 and change the config to new format. Everything works well except site search.  

When `jp` search is configured, 3 Uncaught TypeError happened.

```
Uncaught TypeError: Cannot set property 'stemmerSupport' of undefined
    at lunr.stemmer.support.js:1
    at lunr.stemmer.support.js:1
    at lunr.stemmer.support.js:1
tinyseg.js:1 Uncaught TypeError: Cannot set property 'TinySegmenter' of undefined
    at tinyseg.js:1
    at tinyseg.js:1
    at tinyseg.js:1
lunr.jp.js:1 Uncaught Error: Lunr is not present. Please include / require Lunr before this script.
    at lunr.jp.js:1
    at lunr.jp.js:1
    at lunr.jp.js:1
``` 

Since the code has been compressed, I format them and find that problem may relevant with
` _.TinySegmenter = t` in `tinyseg.js` OR `r.stemmerSupport = { ` in ·lunr.stemmer.support.js`

But I cannot see any `TypeError`, any idea?

**Comment by squidfunk** [comment]
Will investigate, thanks for reporting!

**Comment by squidfunk** [comment]
Fixed in #607

**Comment by squidfunk** [comment]
Released as part of 2.2.0.
